### PR TITLE
Fix UG & DG

### PR DESCRIPTION
**Pull request by Goh-Li-Ting**
View UG and DG here: https://goh-li-ting.github.io/tp/

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -464,7 +464,8 @@ EmployeeManager data are saved in the hard disk automatically after any command
 ### Editing the data file
 
 EmployeeManager automatically saves your data as a JSON file `[JAR file location]/data/employeemanager.json`. 
-For advanced users, you have the option to directly modify this data file to update records. After editing, simply restart Employeemanager to view the update changes.
+For advanced users, you have the option to directly modify this data file to update records.
+After editing, simply restart EmployeeManager to view the update changes.
```
Comment (aexolate) [inline docs/UserGuide.md:468]:
```suggestion
After editing, simply restart EmployeeManager to view the updated changes.
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -742,6 +742,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 * **Mainstream OS**: Windows, Linux, Unix, OS-X.
 * **UI**: User Interface, the graphical or textual interface through which a user interacts with a software application.
 * **Git Miletone**: A git milestone is a way to track progress on a group of issues or pull requests in a repository which can achieved within a specified amount of time by setting a start and due date.
```
Comment (Chen-Kuei) [inline docs/DeveloperGuide.md:744]: should be **Git Milestone** instead of Git Miletone here

**Review by Weiennn — APPROVED** [review]
LGTM

**Review by aexolate — APPROVED** [review]
LGTM

**Pull request merged**